zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/DecisionHandler.java

```diff
@@ -71,6 +71,6 @@ public void updateEntity(
 
   @Override
   public void flush(final DecisionDefinitionEntity entity, final BatchRequest batchRequest) {
-    batchRequest.addWithId(indexName, String.valueOf(entity.getKey()), entity);
+    batchRequest.addWithId(indexName, entity.getId(), entity);
```

Comment:
The `addwithId` with the `entity.getId` as parameter, is the same ass the plain `batchRequest.add`. Could be used to avoid confusion.